### DISCUSSION TITLE
Daily 27/10

### DISCUSSION THREAD

**Discussion by rpgrca** [rpgrca/proyecto-ispc-fullstack#240] [General]
Ayer dividí lo que era lote de subasta así que ahora sí tenemos controlador y modelo de base de datos para lotes. También empecé a armar pruebas para la clase de pujas que subieron y me di cuenta que la cobertura está mal, está haciéndolo sobre los archivos de testing y no sobre el código real, así que voy a tratar de arreglar eso. También voy a seguir agregando más testeos, mirar los dos sincrónicos hoy y tratar de emprolijar los controladores y el modelo en base de datos.

**Comment by NataliaAlvarezIspc** [comment]
Ayer terminé todos los queries que tenía pendiente y los subí al repositorio. Me asigné dos tareas mas de queries y otra de python ya que la que había elegido, cuando actualice el repo ya estaba hecha. Estuve haciendo repaso de Front para la evaluación
Hoy voy a hacer las tareas de sql y entre hoy y mañana la de python, ya que nos dimos hasta el domingo para terminarlas. También voy seguir repasando para la evaluación de Front y leyendo el material para la clase de SGC de hoy. Por la tarde-noche participaré de las clases sincrónicas de SGC y Full.

**Comment by AhumadaCeleste** [comment]
Ayer estudie front, participe de la clase de ingles y complete la lección 5 de imperativo. Hoy quiero resolver una dificultad de subir imágenes a wordpres para comenzar con lo de Almoneda. Los jueves no dispongo de mucho tiempo libre.  El viernes veré las clases de SGC y full y espero poder iniciar con las tareas asignadas del proyecto.

**Comment by Brian-Almada** [comment]
Ayer completé algunas issues que me asigné. Ahora estoy ideando un botón de scroll-top en JS para meter en la parte de catálogo. Veré si me sale.
